_posts/0000-01-11-JackZhong4Openlane.md

```diff
@@ -5,5 +5,7 @@ title: "Welcome to our slide deck!"
 
 Caption here
```

Comment:
You should have put the caption here instead

Reply:
Victor , please review and approve my request.